.github/workflows/expotools.yml

```diff
@@ -32,7 +32,10 @@ jobs:
         run: yarn install --ignore-scripts --frozen-lockfile
         working-directory: tools
       - name: 🛠 Compile TypeScript sources
-        run: yarn tsc
+        run: yarn build
+        working-directory: tools
+      - name: 🛠 Typecheck sources
+        run: yarn tsc --noEmit
```

Comment:
Maybe we should type check before build? 

Not sure which tool errors are better for DX, but if build fails we will see only SWC ones. Swap will make TSC errors only present.

Reply:
tsc takes longer to error